wpilibNewCommands/src/generate/main/native/include/frc2/command/button/commandhid.h.jinja

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+// THIS FILE WAS AUTO-GENERATED BY ./wpilibNewCommands/generate_hids.py. DO NOT MODIFY
+
+#pragma once
+#include <frc/{{ ConsoleName }}Controller.h>
+
+#include "frc2/command/CommandScheduler.h"
+#include "frc2/command/button/Trigger.h"
+
+namespace frc2 {
+/**
+ * A version of {@link {{ ConsoleName }}Controller} with {@link Trigger} factories for
+ * command-based.
+ *
+ * @see {{ ConsoleName }}Controller
+ */
+class Command{{ ConsoleName }}Controller : public frc::{{ ConsoleName }}Controller {
+ public:
+  using {{ ConsoleName }}Controller::{{ ConsoleName }}Controller;
+
+  /**
+   * Constructs an event instance around this button's digital signal.
```

Comment:
The "event instance" references in the command HID classes should be references to "trigger" instead. (Applies generally)